cmd/sonobuoy/app/args.go

```diff
@@ -70,8 +78,9 @@ func AddSonobuoyConfigFlag(cfg *SonobuoyConfig, flags *pflag.FlagSet) {
 }
 
 const (
-	e2eFocusFlag = "e2e-focus"
-	e2eSkipFlag  = "e2e-skip"
+	e2eFocusFlag    = "e2e-focus"
+	e2eSkipFlag     = "e2e-skip"
+	e2eParallelFlag = "e2e-parallel"
 )
 
 // AddE2EConfigFlags adds two arguments: --e2e-focus and --e2e-skip. These are not taken as pointers, as they are only used by GetE2EConfig. Instead, they are returned as a Flagset which should be passed to GetE2EConfig. The returned flagset will be added to the passed in flag set.
```

Comment:
please update the godoc to reflect the new flag and comment on its hidden status and why it is hidden in the UI

Reply:
Done.